oracle-plugin/src/e2e-test/java/io.cdap.plugin/common.stepsdesign/TestSetupHooks.java

```diff
@@ -416,4 +420,25 @@ public static void dropOracleTargetDateTable() throws SQLException, ClassNotFoun
     BeforeActions.scenario.write("Oracle Target Table - " + PluginPropertyUtils.pluginProp("targetTable")
                                    + " deleted successfully");
   }
+
+  @Before(order = 1, value = "@CONNECTION")
+  public static void setNewConnectionName() {
+    String connectionName = "Oracle" + RandomStringUtils.randomAlphanumeric(10);
```

Comment:
This is a common class, we should take connectionPrefix as parameterized, it would not necessary be `Oracle` for all plugins.

Reply:
We are having different TestSetupHooks class file for different database plugins. For each database plugin we are using according to the plugin name.